mkdocs.yml

```diff
@@ -139,113 +142,121 @@ theme:
-          #  - LiteLLM: trulens_eval/api/provider/litellm.md
-          #  - 🦜️🔗 Langchain: trulens_eval/api/provider/langchain.md
-          #- 🤗 HuggingFace: trulens_eval/api/provider/huggingface.md
-      - Feedback Implementations:
-          - 📖 Stock Feedback Functions: trulens_eval/api/providers.md
-          - trulens_eval/custom_feedback_functions.ipynb
-      - Generating Test Cases: trulens_eval/generate_test_cases.md
-      - Feedback Evaluations:
-          # Titles come from notebook and will be overridden if specified here.
-          - trulens_eval/answer_relevance_smoke_tests.ipynb
-          - trulens_eval/context_relevance_smoke_tests.ipynb
-          - trulens_eval/groundedness_smoke_tests.ipynb
+    # PLACEHOLDER: - trulens_eval/evaluation/index.md
+    - Feedback Functions:
+      - trulens_eval/evaluation/feedback_functions/index.md
+      - Anatomy of a Feedback Function: trulens_eval/evaluation/feedback_functions/anatomy.md
+      - Specification Guide: trulens_eval/evaluation/feedback_functions/guide.md
+    - Running Feedback Functions:
+      # PLACEHOLDER: - trulens_eval/evaluation/running_feedback_functions/index.md
+      - Running with your app: trulens_eval/evaluation/running_feedback_functions/with_app.md
+      - Running on existing data: trulens_eval/evaluation/running_feedback_functions/existing_data.md
+    - Feedback Selectors:
+      - trulens_eval/evaluation/feedback_selectors/index.md
+      - Selecting Components: trulens_eval/evaluation/feedback_selectors/selecting_components.md
+      - Shortcuts: trulens_eval/evaluation/feedback_selectors/selector_shortcuts.md
+    - Feedback Implementations:
+      - trulens_eval/evaluation/feedback_implementations/index.md
+      - 🧰 Stock Feedback Functions: trulens_eval/evaluation/feedback_implementations/stock.md
+      - trulens_eval/evaluation/feedback_implementations/custom_feedback_functions.ipynb
+    - Generating Test Cases: trulens_eval/evaluation/generate_test_cases.md
+    - Feedback Evaluations:
+      # Titles come from notebook and will be overridden if specified here.
+      # PLACEHOLDER: - trulens_eval/evaluation/feedback_evaluations/index.md
+      - trulens_eval/evaluation/feedback_evaluations/answer_relevance_smoke_tests.ipynb
+      - trulens_eval/evaluation/feedback_evaluations/context_relevance_smoke_tests.ipynb
+      - trulens_eval/evaluation/feedback_evaluations/groundedness_smoke_tests.ipynb
   - 🎺 Tracking:
-      - Instrumentation:
-          # Titles set inside notebooks and will be overridden if provider here.
-          - trulens_eval/trulens_instrumentation.ipynb
-          - trulens_eval/langchain_instrumentation.ipynb
-          - trulens_eval/llama_index_instrumentation.ipynb
-      - Logging:
-          - Where to Log: trulens_eval/where_to_log.md
-          - 📓 Logging Methods: trulens_eval/logging.ipynb
+    # PLACEHOLDER: - trulens_eval/tracking/index.md
+    - Instrumentation:
```

Comment:
Instrumentation overview is now just accessible from the header which is non-obvious. Can you re-add it as a page under the Instrumentation header?

![Screenshot 2024-03-01 at 3 03 29 PM](https://github.com/truera/trulens/assets/60949774/9fc4256f-a66d-44ec-a038-27030ff102c4)

Reply:
Instrumentation heading/overview is the first page in the tracking section. It is impossible avoiding getting on this page. Also, part of the point of this PR is to move overviews to the headings. Could rename the page as "Instrumentation Overview" to make it more obvious.